### PR TITLE
Add more mkv tags

### PR DESCRIPTION
**Pull request by invliD**
I know ebml-go doesn't explicitly support mkv files, but with these tags added I can successfully parse the ones I care about.

### CODE REVIEW

File: elementtable.go [hunk 1]
```diff
@@ -55,37 +56,49 @@ var table = elementTable{
...
 	ElementCueTrackPositions:      elementDef{[]byte{0xB7}, DataTypeMaster, false},
+	ElementFlagEnabled:            elementDef{[]byte{0xB9}, DataTypeUInt, false},
 	ElementPixelHeight:            elementDef{[]byte{0xBA}, DataTypeUInt, false},
 	ElementCuePoint:               elementDef{[]byte{0xBB}, DataTypeMaster, false},
 	ElementTrackNumber:            elementDef{[]byte{0xD7}, DataTypeUInt, false},
 	ElementVideo:                  elementDef{[]byte{0xE0}, DataTypeMaster, false},
 	ElementAudio:                  elementDef{[]byte{0xE1}, DataTypeMaster, false},
 	ElementTimecode:               elementDef{[]byte{0xE7}, DataTypeUInt, false},
-	ElementVoid:                   elementDef{[]byte{0xEC}, DataTypeMaster, false},
+	ElementVoid:                   elementDef{[]byte{0xEC}, DataTypeBinary, false},
```
Comment (invliD) [inline elementtable.go:100]: The void element is binary: [spec](https://github.com/ietf-wg-cellar/ebml-specification/blob/a61dd6f88a7701dc27c27a5a79b3c187fd2159bb/specification.markdown#void-element)

File: elementtype.go [hunk 2]
```diff
@@ -37,6 +37,7 @@ const (
 	ElementEBMLDocTypeVersion
 	ElementEBMLDocTypeReadVersion
 
+	ElementCRC32
```
Comment (invliD) [inline elementtype.go:40]: I did not attempt to implement any kind of CRC verification. Merely adding the element type allows parsing a file containing them, so I believe this is a good first step.

**Review by at-wat — APPROVED** [review]
LGTM
Thank you for your contribution!

**Comment by at-wat** [comment]
Added v0.12.0 tag

**Pull request merged**